src/MovingHarmonicPotential.cc

```diff
@@ -3,50 +3,45 @@
 // Part of azplugins, released under the BSD 3-Clause License.
 
 /*!
- * \file ImplicitEvaporator.cc
- * \brief Definition of ImplicitEvaporator
+ * \file MovingHarmonicPotential.cc
+ * \brief Definition of MovingHarmonicPotential
  */
 
-#include "ImplicitEvaporator.h"
+#include "MovingHarmonicPotential.h"
+
+namespace hoomd
+    {
 
 namespace azplugins
     {
 /*!
  * \param sysdef System definition
  * \param interf Position of the interface
  */
-ImplicitEvaporator::ImplicitEvaporator(std::shared_ptr<SystemDefinition> sysdef,
-                                       std::shared_ptr<Variant> interf)
+MovingHarmonicPotential::MovingHarmonicPotential(std::shared_ptr<SystemDefinition> sysdef,
+                                                 std::shared_ptr<Variant> interf)
     : ForceCompute(sysdef), m_interf(interf), m_has_warned(false)
     {
     // allocate memory per type for parameters
     GPUArray<Scalar4> params(m_pdata->getNTypes(), m_exec_conf);
     m_params.swap(params);
-
-    // connect to type change to resize type data arrays
-    m_pdata->getNumTypesChangeSignal()
-        .connect<ImplicitEvaporator, &ImplicitEvaporator::reallocateParams>(this);
     }
 
-ImplicitEvaporator::~ImplicitEvaporator()
-    {
-    m_pdata->getNumTypesChangeSignal()
-        .disconnect<ImplicitEvaporator, &ImplicitEvaporator::reallocateParams>(this);
-    }
+MovingHarmonicPotential::~MovingHarmonicPotential() { };
 
 /*!
  * \param timestep Current timestep
  *
  * This method only checks for warnings about the virial.
  * Deriving classes should implement the actual force compute method.
  */
-void ImplicitEvaporator::computeForces(unsigned int timestep)
+void MovingHarmonicPotential::computeForces(unsigned int timestep)
     {
     PDataFlags flags = m_pdata->getFlags();
     if (!m_has_warned
-        && (flags[pdata_flag::pressure_tensor] || flags[pdata_flag::isotropic_virial]))
+        && flags[pdata_flag::pressure_tensor])
```

Comment:
This is correct! See here:
https://github.com/glotzerlab/hoomd-blue/blob/58b7ba503cadd90cc1ed65f40208a8d2de5b7b63/hoomd/ParticleData.h#L68-L78